package.json

```diff
@@ -31,7 +31,20 @@
     "extend": "^3.0.1",
     "message-box": "^0.1.1",
     "mongo-object": "^0.0.3",
-    "underscore": "^1.8.3"
+    "underscore": "^1.8.3",
```

Comment:
The whole point of the PR is removing this dependency, so make sure it isn't imported anywhere and then remove this line.